Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+
+#Troubleshooting
```

Comment:
Double check the Heading - it does not seem to be rendering properly as Heading 1 (at least in the preview).